contrib/test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -ex
 
+set -e
```

Comment:
Being confused about how this script ever worked, I did some digging and saw that this script is actually launched with `-e`. See the shebang line :)

This makes me wonder, this actually change something? Did you test whether or not this makes a difference?

Reply:
Made no difference, wasm-pack errors are swallowed on Mac OS ci